Client/mods/deathmatch/logic/luadefs/CLuaPlayerDefs.cpp

```diff
@@ -636,3 +639,8 @@ unsigned char CLuaPlayerDefs::GetPlayerMapOpacity()
     int iMapOpacity = g_pCore->GetCVars()->GetValue<int>("mapalpha");
     return static_cast<unsigned char>(Clamp(0, iMapOpacity, 255));
 }
+
+bool CLuaPlayerDefs::IsPlayerCrosshairVisible()
```

Comment:
If isCrosshairVisible can be noexcept, mark this lua def as noexcept too, please. Otherwise ignore this comment